docs/farming.md

```diff
@@ -188,6 +188,80 @@ After this, simply repeat the step you prompted for (step 4 or 6). This time, cl
 ```
 7. It may prompt again in here. Refer to the note on step 4.
 
+## Docker Instructions
+
+Create `subspace` directory and `docker-compose.yml` in it with following contents:
+```yml
+version: "3.7"
+services:
+  node:
+    # Replace `snapshot-DATE` with latest release
+    image: ghcr.io/subspace/node:snapshot-DATE
+    volumes:
+# Instead of specifying volume (which will store data in `/var/lib/docker`), you can
+# alternatively specify path to the directory where files will be stored, just make
+# sure everyone is allowed to write there
+      - node-data:/var/subspace:rw
+#      - /path/to/subspace-node:/var/subspace:rw
+    ports:
+# If port 30333 is already occupied by another Substrate-based node, replace all
+# occurrences of `30333` in this file with another value
+      - "0.0.0.0:30333:30333"
+# Un-comment following line to unlock node's WebSocket RPC
+#      - "127.0.0.1:9944:9944"
```

Comment:
Just run configuration with those lines uncommented. It seem to deny connection from farmer as it is another host :)
Should we just open it also for `0.0.0.0`?

Reply:
`0.0.0.0` there determines on which of host's interfaces will it listen for connections. If you run farmer on the same machine it should work with `127.0.0.1` and there should be no need to expose it to public. If you connect from another machine you may need to change it to `0.0.0.0` indeed, but it is not what we officially advertise.

What did you do exactly here?

Reply:
Oh, just rechecked everything is fine :)